pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/cpd/testdata/sample-plsql_ignore-literals.txt

```diff
@@ -0,0 +1,1729 @@
+    [Image] or [Truncated image[            Bcol      Ecol
+L1
+    [CREATE]                                1         7
+    [OR]                                    8         10
+    [REPLACE]                               11        18
+L2
+    [PACKAGE]                               1         8
+    ["TEST_SCHEMA"]                         9         22
+    [.]                                     22        23
+    ["BANK_DATA"]                           23        34
+L3
+    [IS]                                    1         3
+L19
+    [PI]                                    1         3
+    [CONSTANT]                              5         13
+    [NUMBER]                                14        20
+    [:]                                     21        22
+    [=]                                     22        23
+    [<UNSIGNED_NUMERIC_LITERAL>]            24        30
+    [;]                                     30        31
+L20
+    [C]                                     1         2
+    [CONSTANT]                              5         13
+    [NUMBER]                                14        20
+    [:]                                     21        22
+    [=]                                     22        23
+    [<UNSIGNED_NUMERIC_LITERAL>]            24        29
+    [;]                                     29        30
+L21
+    [D]                                     1         2
+    [CONSTANT]                              5         13
+    [NUMBER]                                14        20
+    [:]                                     21        22
+    [=]                                     22        23
+    [<UNSIGNED_NUMERIC_LITERAL>]            24        29
+    [;]                                     29        30
+L22
+    [YEAR_CREATED]                          1         13
+    [CONSTANT]                              14        22
+    [NUMBER]                                23        29
+    [:]                                     30        31
+    [=]                                     31        32
+    [<UNSIGNED_NUMERIC_LITERAL>]            33        37
+    [;]                                     37        38
+L23
+    [AUTHOR]                                1         7
+    [CONSTANT]                              9         17
+    [VARCHAR2]                              18        26
+    [(]                                     26        27
+    [<UNSIGNED_NUMERIC_LITERAL>]            27        30
+    [)]                                     30        31
+    [:]                                     32        33
+    [=]                                     33        34
+    ["\\'"]                                 35        58
```

Comment:
Unfortunately string literals don't appear here as `<STRING_LITERAL>` which would be nicer...

Reply:
Yeah that's a javacc problem... I'm not sure we can transform the code to get the nice name but I will look into it